nix/shell.nix

```diff
@@ -1,9 +1,7 @@
-{ pkgs, deku, ligo, npmPackages }:
+{ pkgs, deku, ligo, nodejs ? pkgs.nodejs }:
 pkgs.mkShell {
   shellHook = ''
-    export NODE_PATH=${npmPackages}/node_modules
```

Comment:
this is a slight loss because now we can't run nodejs locally with our node_modules pre-configured. However, I think it's reasonable, and it's easy to add it back if we need it.

Reply:
This is now wrapped into the webpack binary. We can easily revert this if you run it in another way